Projects/hCore/Sources/FeatureFlags/FeatureFlagOctopus.swift

```diff
@@ -131,13 +131,7 @@ public class FeatureFlagsUnleash: FeatureFlags {
         isTiersEnabled = unleashClient.isEnabled(name: enableTiersKey)
         featureFlags[enableTiersKey] = isTiersEnabled
 
-        let paymentTypeKey = "payment_type"
-        let paymentTypeName = unleashClient.getVariant(name: paymentTypeKey).name
-        if paymentTypeName == "adyen" {
-            paymentType = .adyen
-        } else {
-            paymentType = .trustly
-        }
+        paymentType = .trustly
```

Comment:
do we even need to set specific `trustly` when this is the only option we have now?